Set "TC_AWS_REGION" to s3_aws_region in thumbor.conf.j2

DavisRayM opened the pull request.

DavisRayM committed:
Set "TC_AWS_REGION" to s3_aws_region in thumbor.conf.j2

morrismukiri commented on templates/thumbor.conf.j2:
```diff
@@ -469,6 +469,9 @@ UPLOAD_PUT_ALLOWED = {{ thumbor_upload_put_allowed }}
 ################################################################################
 
 ############################### Thumbor AWS ####################################
+{% if s3_aws_region is defined %}
```
What do you think of adding the `thumbor_` prefix to the variable name?

DavisRayM replied:
I think that'll be nice to have. I'll update the variable

DavisRayM committed:
Update s3_aws_region and s3_create_bucket variable names

- Change s3_aws_region to thumbor_s3_aws_region
- Change s3_create_bucket to thumbor_s3_create_bucket
